indico/modules/attachments/templates/_attachments.html

```diff
@@ -143,9 +143,9 @@
     {%- endif -%}
     {% if management %}
         <div class="i-box-footer text-right js-attachment-actions hide-if-locked">
-            <a href="#" class="i-button icon-folder-plus js-dialog-action"
+            <button class="i-button icon-folder-plus js-dialog-action"
                data-href="{{ url_for('attachments.create_folder', linked_object) }}"
-               data-title="{% trans %}Create a new folder{% endtrans %}">{% trans %}New folder{% endtrans -%}</a>
+               data-title="{% trans %}Create a new folder{% endtrans %}">{% trans %}New folder{% endtrans -%}</button>
```

Comment:
Can you please fix the indentation here (and in other places if necessary)? We typically align the attributes below the first attribute, ie like this:

```html
<button class="..."
        foo="bar">
```